src/backend/joanie/payment/backends/lyra/__init__.py

```diff
@@ -342,16 +339,8 @@ def handle_notification(self, request):
         if not self._check_hash(post_data):
             raise exceptions.ParseNotificationFailed()
 
-        if settings.DEBUG:
-            with open("request.json", "w", encoding="utf-8") as f:
-                json.dump(post_data, f, indent=4)
-
```

Comment:
What do you remove this ? It's helpful for development.

Reply:
As we said friday, we can remove them:
This is a bit hacky, and could never work on prod (even if debug shouldn't be enabled there).
If we want something similar in local, we could do something more elegant with loggers, IMO.